llvm/utils/lit/tests/Inputs/shtest-env-positive/env-temp-redirect.txt

```diff
@@ -0,0 +1,7 @@
+## Test the env command with output redirection to a file.
+# RUN: rm -f %t
+# RUN: env > %t
```

Comment:
I can see from the code that the redirect is only consumed if there are no arguments, but could we also check that `env VAR=value > %t` and `env A=B echo "no env output here please"` behave as expected?